_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -0,0 +1,5162 @@
+	pixel_x = 15
+	},
+/turf/open/space/basic,
+/area/template_noop)
+"qb" = (
+/obj/effect/turf_decal{
+	dir = 8
+	},
+/obj/effect/turf_decal{
+	dir = 1
+	},
+/obj/effect/turf_decal{
+	dir = 8
+	},
+/obj/effect/turf_decal{
+	dir = 9
+	},
+/obj/structure/tank_dispenser{
+	oxygentanks = 0;
+	plasmatanks = 0
+	},
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"qq" = (
+/obj/machinery/light/small/red/dim/directional/east,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white"
+	},
+/obj/item/ammo_casing/spent,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 4
+	},
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/obj/machinery/atmospherics/components/unary/vent_pump/on,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"qu" = (
+/obj/effect/decal/cleanable/blood{
+	icon_state = "floor7-old"
+	},
+/obj/effect/decal/cleanable/glass,
+/obj/machinery/button/door/directional/north{
+	name = "prisoner isolation shutter";
+	id = "captivity";
+	layer = 3.4
+	},
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"qC" = (
+/obj/effect/turf_decal/tile/bar{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/bar,
+/obj/structure/table/wood,
+/obj/item/reagent_containers/cup/glass/bottle/beer{
+	pixel_x = 6;
+	pixel_y = 17
+	},
+/obj/effect/turf_decal{
+	icon_state = "warninglinecorner_white";
+	dir = 4
+	},
+/obj/effect/spawner/random/food_or_drink/dinner{
+	pixel_x = -8;
+	pixel_y = 9
+	},
+/obj/effect/spawner/random/contraband/narcotics{
+	pixel_x = -6;
+	pixel_y = 11
+	},
+/obj/effect/spawner/random/food_or_drink/dinner{
+	pixel_x = 6;
+	pixel_y = 1
+	},
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"qF" = (
+/obj/item/ammo_casing/spent,
+/turf/open/space/basic,
+/area/template_noop)
+"qJ" = (
+/obj/structure/closet/crate/trashcart,
+/obj/item/clothing/mask/facehugger/toy,
+/obj/item/storage/fancy/cigarettes/cigpack_cannabis,
+/obj/effect/mob_spawn/corpse/human/charredskeleton,
+/turf/open/space/basic,
+/area/template_noop)
+"qR" = (
+/obj/effect/decal/cleanable/blood/gibs/old,
+/obj/effect/mob_spawn/corpse/human/syndicatecommando/lessenedgear,
+/turf/open/floor/mineral/plastitanium/airless,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"rd" = (
+/obj/effect/decal/cleanable/blood/gibs/old{
+	icon_state = "gibmid1"
+	},
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/alien/weeds,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"rm" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"rs" = (
+/obj/structure/sign/warning/no_smoking{
+	pixel_x = 28
+	},
+/obj/item/storage/toolbox/syndicate,
+/obj/effect/spawner/random/exotic/tool,
```

Comment:
Don't really like the idea of spawning in alien tools here, especially since it's not an alien ship.